### ISSUE TITLE
[White Label - Audiocard]: Keyboard focus is not moving to the Volume and more control which is present in the chat screen.

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5709]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Users who rely on keyboard‑only navigation, are unable to access or operate essential chat controls. This prevents users from adjusting volume or accessing additional chat options, creating a functional accessibility barrier. The issue impacts usability and results in non‑compliance with keyboard accessibility requirements, potentially blocking affected users from fully interacting with the chat experience.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Focus management, Keyboard navigation

### How to reproduce the issue?

Repro Steps:

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to edit field and provide query 'Audiocard'.
4. Navigate to Audio control.
5. Observe an issue that Keyboard focus is not moving to the Volume and more control which is present in the chat screen.

### What do you expect?

Keyboard focus should sequentially move to Volume control and More controls when navigating using Tab / Shift + Tab. All interactive controls on the chat screen should be reachable and operable using the keyboard.

### What actually happened?

Keyboard focus is not moving to the Volume and more control which is present in the chat screen.

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/keyboard

### Adaptive Card JSON

```json

```
